addon/menu.py

```diff
@@ -159,3 +180,144 @@ def check_correct_game_in_menu(menu_actions_by_game_id, load_current_game_id):
+        profile_controller.get_achievements_repo().clear_achievements()
+
+        msg3 = QMessageBox()
+        msg3.setText(claim_items_deleted_text)
+        msg3.setWindowTitle(claim_items_name)
+        msg3.setStandardButtons(QMessageBox.StandardButton.Ok)
+        msg3.exec() 
+
+SPECIAL_ITEMS = ["healing_potion_ii", "random_firework"]
+def get_special_item_command(id, count):
+    username = local_conf["minecraft_username"]
+    final_count = 0 if count <= 6400 else count - 6400
+    cmd_count = min(count, 6400)
+    
+    if(id == "healing_potion_ii"):
+        # recalulate the max counts because minecraft wont let you /give more than 100 potions
+        # good luck to anyone who has earned thousands of potions because they're gonna have to
+        # run alot of commands
+        final_count = 0 if count <= 100 else count - 100
+        cmd_count = min(count, 100)
+
+        # Supports bedrock
+        if(local_conf["is_bedrock_edition"] == "true"):
+            return f'/give {username} splash_potion {cmd_count} 22', final_count
+
+        # Supports 1.20.6
+        if(local_conf["minecraft_version_is_pre_1_20_5"] == "false"):
+            return f'/give {username} splash_potion[potion_contents={{potion:"strong_healing"}}] {cmd_count}', final_count
+        
+        # Supports between 1.13 and 1.20.5
+        return f'/give {username} splash_potion{{Potion:"minecraft:strong_healing"}} {cmd_count}', final_count
+    
+    if(id == "random_firework"):
+        # random varibles
+        colour = random.randrange(0, 16777215) # random hex colour in decimal form
+        fade_colour = random.randrange(0, 16777215) # random hex colour in decimal form
+        flight_time = random.randrange(0, 5) # random flight time
+        shape = random.randrange(0, 4)
+        has_trail = random.choice([0, 1])
+        has_twinkle = random.choice([0, 1])
+
+        # Supports bedrock
+        if(local_conf["is_bedrock_edition"] == "true"):
+            return f'/give {username} firework_rocket {cmd_count} 0', final_count
+
+        # Supports 1.20.6
+        if(local_conf["minecraft_version_is_pre_1_20_5"] == "false"):
+            shape = random.choice(["small_ball", "large_ball", "star", "creeper", "burst"])
+            has_trail = random.choice(["true", "false"])
+            has_twinkle = random.choice(["true", "false"])
+
+            return f'/give {username} firework_rocket[fireworks={{explosions:[{{shape:"{shape}",colors:[{colour}],colors:[{fade_colour}],has_trail:{has_trail},has_twinkle:{has_twinkle}}}],flight_duration:{flight_time}}}] {cmd_count}', final_count
+        
+        return f'/give {username} firework_rocket{{Fireworks:{{Flight:{flight_time},Explosions:[{{Type:{shape},Flicker:{has_twinkle}b,Trail:{has_trail}b,Colors:[I{colour}],FadeColors:[I{fade_colour}]}}]}}}} {cmd_count}', final_count
+
+        # I cannot stress how annoyed i was to find out the minecraft changed the way you give items with nbt data a month ago
```

Comment:
I didn't know that there was a change to the NBT system, and I'm sorry to hear that...